### ISSUE TITLE
Android Build fail after upgrade to 0.61 (packagingOptions)

### ISSUE THREAD

**Issue by gamingumar** [facebook/react-native#26656]
Android Build fails after upgrading to 0.61.1.
The upgrade helper asks to remove these lines and adding these back fixes the build issue.
```
packagingOptions {
        pickFirst '**/armeabi-v7a/libc++_shared.so'
        pickFirst '**/x86/libc++_shared.so'
        pickFirst '**/arm64-v8a/libc++_shared.so'
        pickFirst '**/x86_64/libc++_shared.so'
        pickFirst '**/x86/libjsc.so'
        pickFirst '**/armeabi-v7a/libjsc.so'
    }
```

React Native version:
```
System:
    OS: macOS 10.14.6
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
    Memory: 290.20 MB / 16.00 GB
    Shell: 5.3 - /bin/zsh
  Binaries:
    Node: 10.15.3 - ~/.nvm/versions/node/v10.15.3/bin/node
    Yarn: 1.19.0 - /usr/local/bin/yarn
    npm: 6.4.1 - ~/.nvm/versions/node/v10.15.3/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.0, DriverKit 19.0, macOS 10.15, tvOS 13.0, watchOS 6.0
  IDEs:
    Android Studio: 3.4 AI-183.6156.11.34.5522156
    Xcode: 11.0/11A420a - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0
    react-native: 0.61.1 => 0.61.1
  npmGlobalPackages:
    react-native-cli: 2.0.1
```

## Steps To Reproduce

1. Upgrade to 0.61 from any previous version
2. Build Android

Error in console:
```
* What went wrong:
Execution failed for task ':app:transformNativeLibsWithMergeJniLibsForDebug'.
> More than one file was found with OS independent path 'lib/x86/libc++_shared.so'
```

Snack, code example, screenshot, or link to a repository:

### Additional Console output:

```
BUILD FAILED in 23s

    at checkExecSyncError (child_process.js:616:11)
    at execFileSync (child_process.js:634:13)
    at runOnAllDevices (/.../node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/runOnAllDevices.js:74:39)
    at buildAndRun (/.../node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/index.js:158:41)
    at then.result (/.../node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/index.js:125:12)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by kesha-antonov** [comment]
Same here

**Comment by hamidfzm** [comment]
Reverting back only the following lines temporary resolved my issue:
```
    packagingOptions {
        pickFirst '**/armeabi-v7a/libc++_shared.so'
        pickFirst '**/x86/libc++_shared.so'
        pickFirst '**/arm64-v8a/libc++_shared.so'
        pickFirst '**/x86_64/libc++_shared.so'
    }
```

But as it says in herms release 0.2.1:

> The C++ runtime library is now packaged in a separate AAR to avoid pickFirst nondeterminisms

It seems it may cause some problems in the future.

**Comment by sunnylqm** [comment]
I also met this error. But just after `gradlew clean` it worked (did not add packagingOptions back)

**Comment by mikehardy** [comment]
I'm attempting to follow the ["use your local react-native fork/branch" instructions](https://github.com/facebook/react-native/wiki/Building-from-source) on the 0.61-stable branch, and I ran into this.

In an unclean app build state (it had already built before I altered the branch to use my own fork/branch, and I just added the changes documented above and rebuilt, I got this.

I tried a clean per @sunnylqm and a rebuild and that did not work.

I reverted to the pickFirst strategy above and that did work.

Here's where to put them in the context of the file, in case anyone else needs to try pasting them back in and isn't sure what groovy scope they are in https://github.com/react-native-community/rn-diff-purge/blob/release%2F0.60.6/RnDiffApp/android/app/build.gradle#L180
